sections/featured-product.liquid

```diff
@@ -1424,18 +1424,13 @@
           "value": "lightbox",
           "label": "t:sections.main-product.settings.image_zoom.options__1.label"
```

Comment:
Should we be removing the translations too? 🤔

Reply:
Ideally, we keep the translations. Similar to the dead code, it just makes our lives easier without causing any negatives.

The translation platform should have some memory of previous translations so that won't cause issues with time whenever we revisit it on the next release, BUT translation conflicts are a pain to deal with so let's just keep it there. There's not much negative in having some extra unused strings.